core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search/dot-category-field-search.component.ts

```diff
@@ -41,10 +41,16 @@ export class DotCategoryFieldSearchComponent {
     $isLoading = input<boolean>(false, { alias: 'isLoading' });
 
     constructor() {
+        // Emit the term to search, if the input is empty hide the result.
         this.searchControl.valueChanges
             .pipe(
                 takeUntilDestroyed(),
                 debounceTime(DEBOUNCE_TIME),
+                tap((value: string) => {
```

Comment:
@oidacra Why is this `tap` necessary? I mean, if the `value.length` is 0, then `this.searchControl.setValue('');` is not an infinite loop in this case? because the setValue is a string with length 0?

Reply:
Thanks, Done.